lib/content-services/src/lib/content-metadata/components/content-metadata-card/content-metadata-card.component.html

```diff
@@ -23,24 +23,6 @@
                 data-automation-id="meta-data-card-edit-aspect">
                 <mat-icon>menu</mat-icon>
             </button>
-            <button *ngIf="!readOnly && hasAllowableOperations()"
-                mat-icon-button
-                (click)="toggleEdit()"
-                [attr.title]="'CORE.METADATA.ACTIONS.EDIT' | translate"
-                [attr.aria-label]="'CORE.METADATA.ACCESSIBILITY.EDIT' | translate"
-                data-automation-id="meta-data-card-toggle-edit">
-                <mat-icon>mode_edit</mat-icon>
-            </button>
         </div>
-        <button *ngIf="displayDefaultProperties" mat-button (click)="toggleExpanded()" data-automation-id="meta-data-card-toggle-expand">
-            <ng-container *ngIf="!expanded">
-                <span data-automation-id="meta-data-card-toggle-expand-label">{{ 'ADF_VIEWER.SIDEBAR.METADATA.MORE_INFORMATION' | translate }}</span>
```

Comment:
meta-data-card-toggle-expand-label data-automation-id is still used in some e2e files, please remove it from there as that element does not exist anymore